ipn/ipnlocal/local.go

```diff
@@ -4614,6 +4616,12 @@ func unmapIPPrefixes(ippsList ...[]netip.Prefix) (ret []netip.Prefix) {
 	return ret
 }
 
+var metricAdvertisedRoutes = usermetrics.NewMultiLabelMap[struct{}](
```

Comment:
Perhaps `usermetrics` could export a helper for tracking metrics with no labels, letting users avoid passing an empty struct every time?